interface/super/rules/controllers/detail/view/view.php

```diff
@@ -103,88 +103,98 @@
                                     echo xlt('Alert Type');
                                 }
                                 ?>:</span></td>
-                            <td colspan="3"><?php
+                            <td colspan="3">
+                                <?php
                                     
-                            if ($something) {
-                                if ($rule->hasRuleType(RuleType::from('activealert')) || $rule->hasRuleType(RuleType::from('passivealert'))) {
-                                    $clinical = '1';
-                                }
-                                if ($rule->hasRuleType(RuleType::from('activealert')) && $rule->hasRuleType(RuleType::from('passivealert'))) {
-                                    $timing = "<div>".xlt('This CR has both an')."
+                                    //of course move this somewhere central??
+                                    function xlts($data)
+                                    {
+                                        if ($GLOBALS['language_default'] !='English (Standard)') {
+                                            $data = preg_replace('/<[^>]*>/', '', $data);
+                                            return xlt($data);
+                                        }
+                                        return $data;
+                                    }
+                                    function xlas($data)
+                                    {
+                                        if ($GLOBALS['language_default'] !='English (Standard)') {
+                                            $data = preg_replace('/<[^>]*>/', '', $data);
+                                            return xla($data);
+                                        }
+                                        return $data;
+                                    }
```

Comment:
Lets not do this since will just cause way more potential problems while having only certain things work as expected for English. Will be better to instead make the constants discussed previously with lots of context {{}} in them.

Reply:
If you need me to list more problems with this solution (such as no mechanism to get them to the translation engine, which means they'll never get translated), I'd be glad to do so :)